OpenEdXMobile/res/layout/row_bulk_download.xml

```diff
@@ -14,17 +14,34 @@
         android:visibility="gone"
         tools:visibility="visible">
 
-        <androidx.appcompat.widget.AppCompatImageView
-            android:id="@+id/iv_icon"
-            android:layout_width="@dimen/ic_large"
-            android:layout_height="@dimen/ic_large"
+        <LinearLayout
```

Comment:
better to change `LinearLayout` with `FrameLayout`. Also, remove properties used for `LinearLayout`.